.circleci/config.yml

```diff
@@ -114,7 +114,7 @@ install_dep_190: &install_dep_190
         # check if we have restored venv cache (/home/circleci/venv) correctly, if so, just skip
         if [ -f /home/circleci/venv/check_version.py ]; then python /home/circleci/venv/check_version.py torch eq 1.8 && exit 0; fi
         # start installing
-        pip install --pre --progress-bar off torch==1.9.0.dev20210415+cu101 torchvision==0.10.0.dev20210415+cu101 -f https://download.pytorch.org/whl/nightly/cu101/torch_nightly.html
+        pip install --pre --progress-bar off torch==1.9.0 torchvision==0.10.0 -f https://download.pytorch.org/whl/test/cu111/torch_test.html
```

Comment:
Please bump the number at the bottom of setup.py as well. That way, a new cache will be stored to avoid repeatedly fetch the pip package by the CI tests.